### DISCUSSION TITLE
Reducing Language Barriers in Tech

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [EddieHubCommunity/support#1180] [General]
@Rahma-AlWadhahi asked this [in our community call originally](https://github.com/EddieHubCommunity/support/discussions/1176#discussioncomment-538506), but we wanted to create a separate discussion so we would do a deep-dive. This is a great discussion and we would love to hear thoughts from everyone.

> I am not sure how to formulate this as a question but I was wondering how would we encourage collaboration and communication if there was some kind of a language barrier.

**Comment by eddiejaoude** [comment]
This is such a great topic, and really difficult too! 🤓 

Great question @Rahma-AlWadhahi and thank you @nhcarrigan for highlighting this 👍 

I think we could maybe have a TwitterSpaces discussion on this, I would love to hear other peoples experiences and feedback too. I have some stories to share also

**Reply by naomi-lgbt** [reply]
That sounds like a great idea (hopefully it works for me this time!)

**Reply by Rahma-AlWadhahi** [reply]
That would be insightful. Is TwitterSpaces available everywhere in the world? 🤔 because I've never used it before.

**Reply by eddiejaoude** [reply]
Yeah it should be available for everyone to join - it should appear in fleets in purple
